### ISSUE TITLE
Lost tags

### ISSUE THREAD

**Issue by cjnaz** [xBrowserSync/App#44]
### Steps to reproduce
1.  See below
2.
3.

### Expected behaviour
Tell us what should happen

### Actual behaviour
- Can't connect to an older database
- Can't restore a prior backup.

### Client app information
**App Platform:** (Chrome, Firefox, iOS, etc) 
Chrome and Android

**App Platform Version:** 
1.4.0 

#### Other info
I upgraded the API from 1.1.0 to 1.1.4 and using APP 1.4.0 on Chrome.  Now, all of my bookmarks seem to be there, I created a new sync ID, sync works to the API from several clients including Android, but none have tags.
	
Things I've done/tried:

- I exported the database from mongo before the API upgrade and reloaded it after.  When attempting to connect to the reloaded database sync ID I get the `Ready to upgrade sync?` message followed by `Something went wrong.  If the problem recurs…`., and to the Chrome background.html: `Sync data: {"type":4,"command":1,"deferred":{"promise":{"$$state":{"status":2,"value":{},"processScheduled":false,"pur":true}}}}.`
(The sync ID and encryption passwords are correct since changing the sync ID results in "Invalid Sync ID".)
	
- I have a dated backup from 1.1.0/1.3.1 which has tags (I see them in the .json backup file).  Attempting to restore this backup on 1.1.4/1.4.0 has a `Working on it… Don't close this window yet.` which doesn't go away (waited 5 minutes).
	
	background.html log output:
```
Error in event handler for (unknown): TypeError: Cannot read property 'children' of undefined
   at Object.upgradeContainers [as UpgradeContainers] (chrome-extension://lcbjdhceifofjlpecfpeimnnphbcjgnc/js/background.min.js:2433:54)
    at chrome-extension://lcbjdhceifofjlpecfpeimnnphbcjgnc/js/background.min.js:3658:47
    at p (chrome-extension://lcbjdhceifofjlpecfpeimnnphbcjgnc/js/background.min.js:142:60)
    at restoreBookmarks (chrome-extension://lcbjdhceifofjlpecfpeimnnphbcjgnc/js/background.min.js:3657:9)
    at handleMessage (chrome-extension://lcbjdhceifofjlpecfpeimnnphbcjgnc/js/background.min.js:3603:13)
    at chrome-extension://lcbjdhceifofjlpecfpeimnnphbcjgnc/js/background.min.js:3613:13
```

Hopefully, both are the same underlying issue, and I can connect to and upgrade my original sync ID which has the current bookmarks with tags.

Thx for your help,
cjn

**Comment by nero120** [comment]
Looks like a bug in the upgrade process, sorry about that @cjnaz. Will look into this.

**Comment by nero120** [comment]
@cjnaz would you be willing to email me (admin@xbrowsersync.org) your xbs backup file so I can debug the restore process?

**Comment by nero120** [comment]
This has been fixed in commit b17d5d7deffff0d5e2c898d3320ff52b8d28355f and will be released in v1.4.1.
